brut.apktool/apktool-lib/src/main/java/brut/androlib/apk/ApkInfo.java

```diff
@@ -295,7 +299,9 @@ public void write(YamlWriter writer) {
         writer.writeBool("resourcesAreCompressed", resourcesAreCompressed);
         writer.writeBool("sharedLibrary", sharedLibrary);
         writer.writeBool("sparseResources", sparseResources);
-        writer.writeStringMap("unknownFiles", unknownFiles);
+        if (unknownFiles.size() > 0) {
```

Comment:
Shouldn't we always write this out even if empty? I know some tools parse the apktool.yml

Reply:
yes this is null and updated in the ApkDecoder.decode but after replace ResUnknownFiles - this field initialized when ApkInfo created. And for store behavior need test empty lines.